docs/configuration/configuration-file.mdx

```diff
@@ -253,16 +479,16 @@ resolve:
       - # header configuration
 ```
 
-The `headers` section supports fields listed in the following table.
+The `headers` property supports the following fields:
 
-| Property | Description | Examples |
+| Field | Description | Examples |
```

Comment:
```suggestion
| Property | Description | Examples |
```
issue: This is to be consistent with the rest of the docs. These are properties (also called fields colloquially).